R/iSEEmarker.R

```diff
@@ -0,0 +1,153 @@
+#' iSEEmarker
+#'
+#' `iSEEmarker()` creates an initial state of an iSEE instance for interactive
+#' exploration of marker genes through the `DynamicMarkerTable` panel, synchronizing
+#' selections with a `ReducedDimensionPlot` and a `FeatureAssayPlot` to visualize
+#' the expression of selected marker genes
+#'
+#' @param sce SingleCellExperiment object
+#' @param reddim_type A string vector containing the dimensionality reduction
+#' @param clusters A character string containing the name of the
+#'   clusters/cell-type/state...(as listed in the colData of the sce)
+#' @param groups A character string of the groups/conditions...(as it appears in
+#'   the colData of the sce)
+#' @param selection_plot_format A string character containing the class of the panel.
+#'  It can be either `ColumnDataPlot` or `ReducedDimensionPlot`
+#'
+#' @return A list of "Panel" objects specifying the initial state of iSEE
+#'   instance
+#' @export
+#' @importFrom methods new
+#' @importFrom SummarizedExperiment colData
+#' @importFrom SingleCellExperiment reducedDimNames
+#' @importFrom methods is
+#' @importClassesFrom iSEE ColumnDataPlot
+#' @importClassesFrom iSEE ReducedDimensionPlot
+#' @importClassesFrom iSEE FeatureAssayPlot
+#' @importFrom BiocBaseUtils isCharacter isScalarCharacter isTRUEorFALSE
+#' @examples
+#' sce <- scRNAseq::RichardTCellData()
+#' sce <- scuttle::logNormCounts(sce)
+#' sce <- scater::runPCA(sce)
+#' sce <- scater::runTSNE(sce)
+#' cluster <- "stimulus"
+#' group <- "single cell quality"
+#' initial <- iSEEmarker(sce = sce, clusters = cluster, groups = group,
+#' selection_plot_format = "ColumnDataPlot")
+#'
+#'
+iSEEmarker <- function(sce,
```

Comment:
nice, nice, nice job!

Shows we have been using this config quite a lot and it seems to be very meaningful!